backend/src/graphql/resolver/ContributionMessageResolver.ts

```diff
@@ -112,18 +122,18 @@ export class ContributionMessageResolver {
       if (!contribution) {
         throw new LogError('Contribution not found', contributionId)
       }
-      if (contribution.userId === user.id) {
+      if (contribution.userId === moderator.id) {
         throw new LogError('Admin can not answer on his own contribution', contributionId)
       }
       if (!contribution.user.emailContact) {
-        contribution.user.emailContact = await UserContact.findOneOrFail({
+        contribution.user.emailContact = await DbUserContact.findOneOrFail({
           where: { id: contribution.user.emailId },
         })
       }
       contributionMessage.contributionId = contributionId
       contributionMessage.createdAt = new Date()
       contributionMessage.message = message
-      contributionMessage.userId = user.id
+      contributionMessage.userId = moderator.id
```

Comment:
if you overwrite the userId with the moderatorId it will cause an inconsistence to the user-object relation.
And below for writing the event it will create a wrong event-entry.

Reply:
![image](https://user-images.githubusercontent.com/1238238/227407356-1ebf62e0-197d-4e6c-adb4-9d42cfc651b3.png)

There is no change here - I just rename `user` to `moderator` for consistency and readability. There is no logic change here. I decided that this was a small enough refactor to be reviewable with big impact.